### ISSUE TITLE
Gaining Experience for Bad Habits

### ISSUE THREAD

**Issue by lapost48** [HabitRPG/habitica#4763]
I am gaining experience and gold when I click on a minus for my habits. I don't know the circumstances leading up to the issue but I will check before I hit the button the next few times and see if I can notice anything obvious.

**Comment by crookedneighbor** [comment]
@lapost48 Is it that you're gaining experience and gold, or that it's syncing past experience and gold (maybe checked off on another computer or mobile device) from the server when you click the minus button?

**Comment by crookedneighbor** [comment]
IE, if you press the minus button repeatedly, do you continue to gain exp and gold?

If so, can you post your user id here? It can be found in [settings > api](https://habitrpg.com/#/options/settings/api) (be sure to copy only the user ID, not the API Token). Also, what is the name of the negative habit that is causing the exp and gold boost?

**Comment by lapost48** [comment]
I think that it is just updating from across systems. I just tested it with my laptop and desktop and it worked fine on a test habit I made but usually I am only using one computer throughout the day. I have a chrome extension that updates my account every 25 minutes so that is probably the source of the mysterious gained gold and exp. I apologize for the fraudulent bug report. I figured I was missing something since other people weren't having the issue and it wouldn't make sense for it to target me.

Thanks and Sorry :disappointed:
